datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BaseE2ETest.java

```diff
@@ -230,18 +224,25 @@ protected UserIDAuth createJohnTestUser(int i) {
         );
     }
 
-    protected void writeDataToFileForUser(UserIDAuth john, String filePathForWriting, byte[] bytes,
+    protected void writeDataToFileForUser(UserIDAuth john, String filePathForWriting, String filePathForReading,
```

Comment:
Consider moving this method to other class that extends BaseE2E and extending from it, specially for concurrency